kong-ngx-build

```diff
@@ -279,7 +279,7 @@ main() {
     notice "Building OpenSSL..."
 
     pushd $OPENSSL_DOWNLOAD
-      if [[ ($OPENSSL_VER == 1.0.* && ! -d include/openssl) || ! -f Makefile ]]; then
+      if (version_lt $OPENSSL_VER 1.1.0 && [[ ! -d include/openssl ]]) || [[ ! -f Makefile ]]; then
```

Comment:
`version_lte $OPENSSL_VER 1.0` reads better - I can update this during the merge.